src/styles/card.scss

```
@@ -6,16 +6,23 @@
   padding: 15px 14px;
   border: 1px solid #f3f3f3;
   border-radius: 5px;
-  
+
   &__image {
-    height: 134px;
-    width: 160px;
-    margin: 15px 5px 0;
+    width: 170px;
+    height: 164px;
+    padding: 12px 2px;
+    transition: transform 1s;
+    border: 3px solid transparent;
```

Пропустив дрібницю з сортуванням
```suggestion
    padding: 12px 2px;

    border: 3px solid transparent;
    transition: transform 1s;
```